tests/Event/EventCommandHandlerTest.php

```diff
@@ -99,7 +99,7 @@ public function it_should_create_a_new_event(): void
         $this->scenario
             ->withAggregateId($id)
             ->when($command)
-            ->then([new EventCreated($id, $language, $title, $type, $location, $calendar, $theme, $now)]);
+            ->then([new EventCreated($id, $language, $title->toString(), $type, $location, $calendar, $theme, $now)]);
```

Comment:
Could make sense here to change the construction of the $`title` to be a primitive instead of the`Title` class?